Client/Frontend/Browser/UserScriptManager.swift

```diff
@@ -475,6 +475,7 @@ class UserScriptManager {
       if !AppConstants.buildChannel.isPublic {
         if let script = walletProviderScript,
            tab?.isPrivate == false,
+           (tab?.secureContentState == .localHost || tab?.secureContentState == .secure),
```

Comment:
Don't think this will be safe to check at this point? It's not guaranteed that the tab has loaded any content at all yet, which would mean this is set to `unknown`?